backend/Dockerfile

```diff
@@ -21,5 +23,7 @@ RUN npm install
 COPY --chown=$UID:$GID [ "./", "/app/" ]
 
 EXPOSE 17442
+# switch back to root to allow UID/GID changing
+USER root
```

Comment:
Interesting, so does entrypoint run as root, but the actual CMD runs as the specified UID/GUID?

Reply:
Also, make sure you run a quick download test as having node_modules not chowned ever may result in the error seen in #72

Reply:
The CMD is passed to the entry point as an argument. Inside the entrypoint we start the server with the specific GID/UID of you pass nothing or an argument starting with ``-``. If you change the CMD to anything else or is run as root to be able to easily debug anything without permission issues. We can adjust this is you like.

Reply:
Ok, understood. Merging...